### DISCUSSION TITLE
Support for .NET 7

### DISCUSSION THREAD

**Discussion by adarcrom** [Azure/azure-mobile-apps#614] [General]
Hello, I downloaded the nuget for this sdk in my .net maui project targeting .net 7. It errors when I build the project with incompatibility issue. I noticed this only supports .net 6. How do I use this package for .net maui project targeting .net 7 and when is this going to be supported. Thank you.

**Comment by adrianhall** [comment]
.NET 7 is backwards compatible with .NET 7.  I haven't tried it, but file an issue and I'll get a fix done.

**Reply by adarcrom** [reply]
Alright will do
